### ISSUE TITLE
Catalog entry without a time axis

### ISSUE THREAD

**Issue by misev** [FAIRiCUBE/catalog#29]
We have some data that is just 2D and has no time axis. But the catalog seems to not allow submitting such a request, this should be fixed.

**Comment by baloola** [comment]
I will work on allowing 2D data, but  the generated stac item must have either begin/end time or a datetime value in order to be ingested  into pgsatc db ( this is why the start/end =  1900/2999 or datetime= 2000 values are assigned to items when no time is specified).

Any suggesgion on how to do it differently ?

**Comment by misev** [comment]
That's unusual, non-timeseries data is pretty common. Could be 2D, or volumetric 3D, etc. I'm not familiar with the catalog standard/spec so don't really have any suggestions. Maybe @KathiSchleidt has some insights.

**Comment by misev** [comment]
> must have either begin/end time or a datetime value

But does it have to be associated with an actual axis? You could put a datetime as a metadata of when the data was created or so, but shouldn't require a time axis for that.

**Comment by baloola** [comment]
> > must have either begin/end time or a datetime value
> 
> But does it have to be associated with an actual axis? You could put a datetime as a metadata of when the data was created or so, but shouldn't require a time axis for that.

yes, providing creation time should suffice, sorry for the confusion!

**Comment by jetschny** [comment]
is this also done and only needs approval?

**Comment by mari-s4e** [comment]
Hi @baloola, so to clarify: when the data refers to a single point in time, you recommend to use the fields Date:Creation Date, instead of the Time axis?

**Comment by baloola** [comment]
> Hi @baloola, so to clarify: when the data refers to a single point in time, you recommend to use the fields Date:Creation Date, instead of the Time axis?

yes

**Comment by KathiSchleidt** [comment]
@mari-s4e I'd worry about using creation date, as that usually refers to the time the dataset was calculated, especially with modeled data this can be years after the fact.

While we could implement a different approach for datasets that cover only one point in time, I do like keeping things consistent. Any reason we cannot just use the Time Axis component, provide same begin and end time for such datasets?

**Comment by baloola** [comment]
is this issue still not resolved?

**Comment by mari-s4e** [comment]
for UC1 datasets, we decided to go with the approach outlined by @KathiSchleidt.
